### DISCUSSION TITLE
Welcome to JetiLuaDFM Discussions!

### DISCUSSION THREAD

**Discussion by davidmcq137** [davidmcq137/JetiLuaDFM#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JoeHitzel4** [comment]
Hello Dave,
I've watched a lot of Harry Curzon's Jeti instructional videos which have been very helpful.  I have successfully been using your air speed Lua App, and it has been helpful.  Currently, I'm trying to load your Virtual SensorXF file to change KMP to MPH for a SM Modell GPS Logger Vario.  Within Harry's Comments, several others comment they have successful accomplished what I'm trying to do.  I've again watched several of Harry Curzon YouTube videos and I've been struggling with trying to load your file for the Virtual SensorXF into my Jeti Studio and I can not seem to have the  Jeti Lua App Manager find it, (after pasting it under File-Configuration, in the Apps Sources Tab).  I have copied the link from Harry's Episode #1 YouTube video (deleting the spaces as Harry instructs) and something I'm doing is not working, of the file is not recognized.  Can you please share the file with me directly? Thanks Dave!

**Comment by davidmcq137** [comment]
Hi. I am traveling right now and all my Jeti stuff is on my linux system at home. Did you put  http:// www.jetiluadfm.app/staticrepo/DFMHC/Apps.json into apps sources on Jeti Studio? Then look for V-SensXF (this one does not follow the naming convention of DFM-XXXX) in the transmitter wizard? It has always been there unless something is awry with my hosting site...

If this does not work for you, I can send you the files when I am back at home .. or if you are good with computer stuff, all my apps are on GitHub: https://github.com/davidmcq137/JetiLuaDFM

**Comment by JoeHitzel4** [comment]
Thanks Dave!!
I was successful, I am not sure what I was doing wrong, however I tried
again yesterday morning and I successfully got the App loaded.  This
morning, I believe I successfully got the programming done, converting kmh
to mph and added a couple of audio files.  Thanks so much for getting back
to me!

**Comment by davidmcq137** [comment]
Glad to hear it! ThanksSent from my iPhoneOn May 13, 2025, at 09:19, JoeHitzel4 ***@***.***> wrote:﻿
Thanks Dave!!

I was successful, I am not sure what I was doing wrong, however I tried

again yesterday morning and I successfully got the App loaded.  This

morning, I believe I successfully got the programming done, converting kmh

to mph and added a couple of audio files.  Thanks so much for getting back

to me!



On Mon, May 12, 2025 at 7:44 PM davidmcq137 ***@***.***>

wrote:



> Hi. I am traveling right now and all my Jeti stuff is on my linux system

> at home. Did you put http:// www.jetiluadfm.app/staticrepo/DFMHC/Apps.json

> into apps sources on Jeti Studio? Then look for V-SensXF (this one does not

> follow the naming convention of DFM-XXXX) in the transmitter wizard? It has

> always been there unless something is awry with my hosting site...

>

> If this does not work for you, I can send you the files when I am back at

> home .. or if you are good with computer stuff, all my apps are on GitHub:

> https://github.com/davidmcq137/JetiLuaDFM

>

>

>

>

>

**Comment by JoeHitzel4** [comment]
Dave,
I have another question for you.  When I am working with you Lua App for
the Maps, I can choose and select Microsoft and their Maps do successfully
load, however I can not seem to get Google to show up. When I select
Google, Google just leaves me with a blank / white screen.  Might you know
what my challenge is?

One other question, much of the Jeti stuff is reporting in Metric.  Using
the Jetibox, I do not see m/s or km/h, however I do not see MPH as an
option.  Is there a solution for this and or do you know if Jeti might
provide those units for us Yankees?  I have with Harry's video, loaded and
used your Virtual Sensor to obtain a work around for telemetry while I am
flying?

What part of the world do you live in? I've lived in the Phoenix metro
area, Arizona, USA since going to ASU in 1978.

If you prefer, my email address is ***@***.***

Thanks Dave
